### ISSUE TITLE
Feedback loop for cargo distance producing odd results for nonroad vehicles

### ISSUE THREAD

**Issue by robbieorvis** [EnergyInnovation/eps-us#117]
I tried testing some very high freight cargo shifting policies and noticed some odd effects as a results of the macroeconomic feedback effects, which grow cargo distance by the change in GDP from ISIC 30 across all nonroad vehicle types. When turning on a high degree of freight mode shifting, this resulted in an enormous increase in _passenger_ air travel with the current design. It seems like we may need a different approach for the feedback here, or at a minimum, shouldn't have changes occurring across cargo types.

**Comment by jrissman** [comment]
The reason why this one happens is pretty straightforward.  ISIC 30 is the ISIC code for manufacturers of non-road vehicles (aircraft, rail, and ship manufacturers).  The macroeconomic feedback for non-road transport demand is based on the percentage growth in value added for ISIC 30.

Shifting freight from HDVs to rail moves value from ISIC 29 (manufacturers of on-road vehicles) to ISIC 30 (manufacturers of non-road vehicles).  That's fine.

But since ISIC 30 is lumped across the non-road modes, when the air demand code checks ISIC 30, it sees an increase, so it ups the air travel demand.  We know the increase in ISIC 30 was entirely due to rail, but it doesn't know that.

A few options are:

- Live with it, as a known source of inaccuracy stemming from our use of 36 ISIC codes, instead of a vastly larger number
- Disable the macroeconomic feedback for non-road travel demand services (but I think this would be undesirable, because it would knock out an effect many people have wanted - that the model now (finally) increases freight travel demand in response to increased industrial goods production)
- Pick some other ISIC code, or a weighted average of ISIC codes, to use to estimate the macroeconomic increase in freight travel demand.  Maybe an output-weighted average of all manufacturing ISIC codes?

**Comment by robbieorvis** [comment]
Are we double counting the effect here, though? Probably not, but just checking.

If we move from freight HDV to freight rail, that moves from ISIC 29 to ISIC 30. The growth in ISIC 30 should already be accounted for from the new travel demand and growth in rail cars. If we cycle that back through again as a multiplier, are we double counting?

Putting that aside though, I think there is a separate issue. ISIC 29 and ISIC 30 are manufacture of on-road and non-road vehicles, but ISIC 49T53 is Transportation and Storage, which is actually where changes in demand for industrial goods should influence freight demand. Changes in the demand for vehicles might affect travel demand, but ISIC 49T53 is probably a better indicator in changes in demand for freight services.

Here is the ISIC code definition for 49T53: “This section includes the provision of passenger or freight transport, whether scheduled or not, by rail, pipeline, road, water or air and associated activities such as terminal and parking facilities, cargo handling, storage etc. Included in this section is the renting of transport equipment with driver or operator. Also included are postal and courier activities”

So I think I would recommend revising the current methodology to use the change in value added from ISIC 49T53 for scaling the freight categories, including both road and non-road (probably just using a single multiplier across both types).

For feedbacks to changes in passenger demand, I think we could possibly calculate those as the percent change in government and household spending on ISIC 19, which covers refined petroleum products.

What do you think?
________________________________
Robbie Orvis
Director of Energy Policy Design
Phone: 415-799-2171
98 Battery Street, Suite 202
San Francisco, CA  94111
www.energyinnovation.org<http://www.energyinnovation.org/>
[cid:image001.jpg@01D0D699.20A24470]
________________________________

Check out our new book, Designing Climate Solutions: A Policy Guide for Low-Carbon Energy<https://www.amazon.com/Designing-Climate-Solutions-Policy-Low-Carbon/dp/1610919564>
Available wherever books are sold

[Policy Design book cover]

**Comment by jrissman** [comment]
No, we're not double counting.  We remove the direct effects on the ISIC codes and only use the indirect and induced effects on the ISIC codes in the energy demand feedback loops.  The only question is which ISIC code(s) to use to estimate the indirect effects on transport demand.

Indirect and induced demand for vehicles seemed like a good way to estimate indirect and induced demand for transportation services.

ISIC 49T53 seems like it might be okay as well.

We cannot separate "government and household spending" on ISIC 19 because we are talking about indirect and induced effects here, which is an output of the I/O model after all first-order changes in output have been cumulated and fed through it.  I doubt using ISIC 19 is reasonable without that limit because there are many sources of demand for petroleum products other than domestic passenger travel.  Exports are a big one.  So is domestic use of petroleum fuels for everything other than passenger transport.

**Comment by jrissman** [comment]
I just noticed that the OECD's [STAN database](https://stats.oecd.org/Index.aspx?DataSetCode=STANI4_2020) breaks out ISIC 49T53 into its components, including putting air transport and water transport services into separate ISIC codes.  So this issue might be able to be resolved cleanly with an ISIC code split.

![StanTransportStorageSplit](https://user-images.githubusercontent.com/7120106/102401880-d1a89f80-3f98-11eb-81f2-094510e5971c.PNG)

This is similar to the chemicals/pharma split, where OECD had this broken out in the STAN database for many countries, but not in the I/O matrices.

**Comment by jrissman** [comment]
In the latest OECD data update (see issue #207), the OECD fully breaks out transportation and storage services, not just in the STAN database but also in the input-output tables.  Therefore, after we move to the updated ISIC codes from OECD, it should be possible to use those new ISIC codes in the transport demand macroeconomic feedback loop to fix the current issue (#117).

**Comment by jrissman** [comment]
In issue #242, Megan wrote:

> While working on the US model, I realized that the Buy In Region lever causes changes in transportation demand, since the macroeconomic feedbacks in the transportation sector are based on changes to ISIC codes 29 and 30. However, just because consumers are buying domestically produced cars rather than imported cars doesn't mean the demand for vehicle-miles should change.

This might be resolved with the new OECD ISIC codes.  We could potentially do a short-term, temporary fix for 3.4 where we base freight transport demand on something like total change in industrial output rather than on vehicle manufacturing.

**Comment by robbieorvis** [comment]
In either instance, passenger LDV travel should be linked household income (ie personal consumption expenditure) changes if it isn’t already (I can’t recall how we are scaling this).

We should double check the new ISIC codes Jeff mentions to make sure the alignment is correct but it sounds like linking to the ISIC code breakdowns from the new IO data should
Solve the long term problem in 3.5.

For freight, because it’s pretty easy to implement the basic fix Megan outlined, I’d recommend we go for it. I already have the data handy for the ISIC code breakdown if we wanted to do that, but it’s probably not strictly necessary. However, given that freight is mostly tied to manufacturing and not services, it might be helpful.

One other thought: isn’t there a transportation and warehousing ISIC code? If so, we could link to that for now I think.


On Jun 9, 2022, at 6:54 PM, Jeff Rissman ***@***.***> wrote:

﻿

In issue #242<https://github.com/Energy-Innovation/eps-us/issues/242>, Megan wrote:

While working on the US model, I realized that the Buy In Region lever causes changes in transportation demand, since the macroeconomic feedbacks in the transportation sector are based on changes to ISIC codes 29 and 30. However, just because consumers are buying domestically produced cars rather than imported cars doesn't mean the demand for vehicle-miles should change.

This might be resolved with the new OECD ISIC codes. We could potentially do a short-term, temporary fix for 3.4 where we base freight transport demand on something like total change in industrial output rather than on vehicle manufacturing.

**Comment by jrissman** [comment]
Fix completed for 3.4.  Leaving this open so we can evaluate whether to update to use one of the OECD's new ISIC codes in 3.5 or whether we want to keep it as is after this fix.  Lowering priority level.

**Comment by jrissman** [comment]
We scheduled the OECD ISIC code update for 3.6 rather than 3.5, so I'm moving this item (which depends on the ISIC code update) to 3.6.

**Comment by robbieorvis** [comment]
This should be addressed as part of updates to issue #299
